src/main.rs

```diff
@@ -1,20 +1,24 @@
 mod database;
 mod git;
+mod github;
 
 use std::path::Path;
 use std::path::PathBuf;
 use structopt::StructOpt;
 
 use crate::database::Database;
 
+const RED: &'static str = "\x1b[1;31m";
+const RESET: &'static str = "\x1b[1;0m";
+
 #[derive(StructOpt)]
 struct Opt {
     #[structopt(subcommand)]
-    command: Command,
+    command: Mode,
 }
 
 #[derive(StructOpt)]
-enum Command {
+enum Mode {
```

Comment:
This is just a hold-over from an earlier change where I needed to run a `Command` using `tokio::process::Command`. Keeping the diff to support this later.